tools/src/h5stat/h5stat.c

```diff
@@ -145,6 +151,17 @@ static H5FD_hdfs_fapl_t hdfs_fa = {
 };
 #endif /* H5_HAVE_LIBHDFS */
 
+static H5FD_onion_fapl_info_t onion_fa_g = {
+    H5FD_ONION_FAPL_INFO_VERSION_CURR,
+    H5P_DEFAULT,                   /* backing_fapl_id                */
+    32,                            /* page_size                      */
+    H5FD_ONION_STORE_TARGET_ONION, /* store_target                   */
+    H5FD_ONION_FAPL_INFO_REVISION_ID_LATEST,
+    0,            /* force_write_open               */
+    0,            /* creation_flags                 */
+    "input file", /* comment                        */
+};
```

Comment:
Does this generate a size warning?

Reply:
Don't know - it was copied from h5dump.c and is used exactly in other tools.

Reply:
So that would be "yes"